src/backends/azure.rs

```diff
@@ -245,55 +245,50 @@ impl Repository for AzureRepository {
             .delimiter(query_delmiter)
             .into_stream();
 
-        if let Some(blob_result) = stream.next().await {
-            match blob_result {
-                Ok(blob) => {
-                    if blob.max_results.is_some() {
-                        result.max_keys = blob.max_results.unwrap() as i64;
-                    }
+        if let Some(Ok(blob)) = stream.next().await {
```

Comment:
I'm not sure that this is exactly equivalent. We used to return an `Err` if `blob` was not `Ok`. We're not anymore.